common/scripts/x86_64/Ubuntu_16.04/remote/unseal_vault.sh

```diff
@@ -1,33 +1,132 @@
 #! /bin/bash
 set -e
 
-export ADMIRAL_ENV="/etc/shippable/admiral.env"
-export BOOT_WAIT=10
-export VAULT_ADDR='http://127.0.0.1:8200'
+readonly VAULT_API_RESPONSE_FILE="/tmp/.vault_response"
```

Comment:
I think we should tell the user about this file if an error happens.

Reply:
done